packages/manager/modules/sms/src/sms/batches/details/delete/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,10 @@
+{
+  "sms_batches_history_details_delete_title": "Confirmation",
+  "sms_batches_history_details_delete_question": "Êtes-vous sûr de vouloir supprimer le SMS affiché ci-dessous ?",
+  "sms_batches_history_details_delete_remove_ko": "Impossible de supprimer le SMS.<br/>{{error}}",
```

Comment:
Is it mandatory to have a new line for the `error` message

Reply:
Copy paste is bad :)